packages/kolibri-common/components/SyncSchedule/EditDeviceSyncSchedule.vue

```diff
@@ -270,15 +270,21 @@
           });
       },
       handleSaveSchedule() {
-        FacilityResource.fetchModel({ id: this.$store.getters.activeFacilityId, force: true }).then(
+        FacilityResource.fetchModel({ id: this.$route.params.facilityId, force: true }).then(
```

Comment:
Relying on the route parameters here might be a bit brittle - may be better to pass in both the facilityId and the deviceId as props to this component, so that we're not reliant on a certain route being in place when this is used.